src/main/java/invtweaks/InvTweaksHandlerShortcuts.java

```diff
@@ -437,16 +437,18 @@ private void moveAll(ShortcutConfig shortcut, ItemStack stackToMatch) {
 
                     // Continue if movement succeeded, there is another slot to try, or we're dropping items.
                     // In reverse: fail if movement failed, AND there are no other slots AND we're not dropping.
-                    if(success || (newIndex != toIndex) || (shortcut.action == ShortcutSpecification.Action.DROP)) {
+                    if (newIndex == toIndex) {
```

Comment:
sorry about the wrong commit title. this patch is actually fix the infinite problem the multi drop is original fix by the toindex == -1 that if condition. but accoring to my recently test, did not found the multi-drop problem. the situation is post on the previous reply